CSS/style.css

```diff
@@ -0,0 +1,179 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
```

Comment:
I think you have to not redeclare if your declare in reset css in your lib.

Reply:
ohh ,ok i will correct it thank you for pointing out